tests/zbd/010

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-3.0+
+# Copyright (C) 2022 Google LLC
+
+. tests/zbd/rc
+. common/null_blk
+. common/scsi_debug
+
+DESCRIPTION="test gap zone support with F2FS"
+QUICK=1
+
+requires() {
+	_have_fio
+	_have_kernel_option BLK_DEV_ZONED
+	_have_modules null_blk
+	_have_module_param scsi_debug zone_cap_mb
+	_have_program mkfs.f2fs
```

Comment:
It would be the better to ensure f2fs module is available here.
```
_have_driver f2fs
```

Reply:
Done :-)